From 04fbb741689d5f25b176c8d9709bfc6d19b2dbb5 Mon Sep 17 00:00:00 2001
Message-Id: <04fbb741689d5f25b176c8d9709bfc6d19b2dbb5.1589921435.git.liu.denton@gmail.com>
In-Reply-To: <cover.1589885479.git.liu.denton@gmail.com>
References: <cover.1589885479.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Tue, 19 May 2020 16:47:17 -0400
Subject: [PATCH v3 8/7] fixup! pkt-line: extern packet_length()
To: Git Mailing List <git@vger.kernel.org>
Cc: Jeff King <peff@peff.net>,
    Eric Sunshine <sunshine@sunshineco.com>,
    Junio C Hamano <gitster@pobox.com>

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 pkt-line.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkt-line.h b/pkt-line.h
index 6cb92d7a5d..5b373fe4cd 100644
--- a/pkt-line.h
+++ b/pkt-line.h
@@ -79,7 +79,7 @@ int packet_read(int fd, char **src_buffer, size_t *src_len, char
  * Convert a four hex digit packet line length header into its numeric
  * representation.
  *
- * If linelen contains non-hex characters, return -1. Otherwise, return the
+ * If lenbuf_hex contains non-hex characters, return -1. Otherwise, return the
  * numeric value of the length header.
  */
 int packet_length(const char lenbuf_hex[4]);
-- 
2.27.0.rc0.114.g141fe7d276

